index.php

```diff
@@ -1,6 +1,11 @@
 <?php
 
-// require_once __DIR__.'/bootstrap.php';
+use Cms\Controllers\Contact;
+use Cms\Controllers\Home;
+
+// require_once("vendor/autoload.php");
```

Comment:
please remove this file all together. `index.php` should not be there since we are relying on routing for all the pages.